gaming-website/package.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "gaming-website-2",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
-  "scripts": {
-    "dev": "vite",
-    "build": "vite build",
-    "preview": "vite preview"
-  },
-  "devDependencies": {
-    "autoprefixer": "^10.4.17",
-    "postcss": "^8.4.35",
-    "tailwindcss": "^3.4.1",
-    "vite": "^5.1.0"
-  }
+	"name": "gaming-website-2",
+	"private": true,
+	"version": "0.0.0",
+	"type": "module",
+	"scripts": {
+		"dev": "vite",
+		"docker-build": "docker-compose up && docker run -p 3000:3000 -d gaming-website",
```

Comment:
You can have the ports and the folder setup straight in the docker compose file
So that you don't need to fiddle with this. 
These are also items that don't change very often (if at all)